plugins/module_utils/network/iosxr/argspec/ospf_interfaces/ospf_interfaces.py

```diff
@@ -311,13 +319,17 @@ def __init__(self, **kwargs):
                         "link_down_fast_detect": {"type": "bool"},
                         "message_digest_key": {
                             "type": "dict",
+                            "no_log": False,
                             "options": {
                                 "id": {"type": "int", "required": True},
                                 "md5": {
                                     "type": "dict",
                                     "required": True,
                                     "options": {
-                                        "password": {"type": "str"},
+                                        "password": {
+                                            "type": "str",
+                                            "no_log": False,
```

Comment:
Should this be True?

Reply:
will address this in https://github.com/ansible-collections/cisco.iosxr/pull/123